lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/mapping/DefaultIdentifierMapping.java

```diff
@@ -11,26 +11,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.plugin.jdbc.mapping;
+package io.trino.plugin.base.mapping;
 
-import com.google.inject.Inject;
 import io.trino.spi.security.ConnectorIdentity;
 
-import java.sql.Connection;
-
 import static java.util.Locale.ENGLISH;
-import static java.util.Objects.requireNonNull;
 
 public class DefaultIdentifierMapping
         implements IdentifierMapping
 {
-    private final RemoteIdentifierSupplier remoteIdentifierSupplier;
-
-    @Inject
-    public DefaultIdentifierMapping(RemoteIdentifierSupplier remoteIdentifierSupplier)
-    {
-        this.remoteIdentifierSupplier = requireNonNull(remoteIdentifierSupplier, "remoteIdentifierSupplier is null");
-    }
+    // Caching this on a field is LazyConnectorFactory friendly
```

Comment:
LazyConnectorFactory?

Reply:
it was comment from previous version, which I reverted
refactored